contracts/airswap/AirSwapFeeConnector.sol

```diff
@@ -0,0 +1,202 @@
+pragma solidity 0.5.17;
+
+import "../openzeppelin/SafeMath.sol";
+import "../openzeppelin/PausableOz.sol";
+import "../openzeppelin/IERC20_.sol";
+
+import "./IAirSwapFeeConnector.sol";
+import "./ISwapERC20.sol";
+
+contract AirSwapFeeConnector is PausableOz, IAirSwapFeeConnector {
+    using SafeMath for uint256;
+
+    struct SwapRequest {
+        address sender;
+        address recipient;
+        uint256 nonce;
+        uint256 expiry;
+        address signerWallet;
+        address signerToken;
+        uint256 signerAmount;
+        address senderToken;
+        uint256 totalSenderAmount;
+        uint8 v;
+        bytes32 r;
+        bytes32 s;
+    }
+
+    uint256 public constant POINTS = 1000;
```

Comment:
By having the points = 1000, this means the smallest fee that we will have is 0.1%, am i correct?